### PR TITLE
Changing controller name to drinks and Homework

### PR DESCRIPTION
**Pull request by ogonzal87**
{
"comfortability": 3,
"completeness": 4,
“comments: “one test is not passing and i am not sure why. I am defining the variable but it keeps saying undefined local variable or method `valid_attributes' for #<RSpec::ExampleGroups::DrinksController::POSTCreate:0x007fe0aec932b8>
     # ./spec/controllers/drinks_controller_spec.rb:84:in`block (3 levels) in <top (required)>"
}

### CODE REVIEW

File: spec/controllers/drinks_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+	    it 'create new drink' do
+	        expect{
+	          post :create, drink: valid_attributes
+	        }.to change(Drink, :count).by(1)
+	      end
+			end
+		
+
+			it 'redirects to drinks#index' do
```
Comment (rgpass) [inline spec/controllers/drinks_controller_spec.rb]: Tabbing

File: app/models/old_drink.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# This class describes a single recipe and its characteristics. 
+# This class describes a single drink and its characteristics. 
 
-class Reciipe
+class Drinkkk
```
Comment (rgpass) [inline app/models/old_drink.rb]: Whoa, racist.

**Comment by rgpass** [comment]
Let's review your error in person then I can give the :+1:

**Comment by ogonzal87** [comment]
@rgpass  :+1:?

**Comment by rgpass** [comment]
:+1:

**Pull request merged**